packages/react-scripts/template-ef/blueprint.yml

```diff
@@ -0,0 +1,44 @@
+version: 1.0
+name: frontier-{{APP_NAME}}
```

Comment:
see if it is easy/quick to replace these variables while I'm testing this branch